test/extended/openstack/cloud-provider-config.go

```diff
@@ -104,6 +104,70 @@ var _ = g.Describe("[sig-installer][Suite:openshift/openstack] The Openshift", f
 				}
 			}
 		})
+
+		//https://bugzilla.redhat.com/show_bug.cgi?id=2074471
+		g.It("should update use-octavia property depending on the NetworkType", func() {
+
+			userConfigMap := config_params{
+				namespace: "openshift-config",
+				name:      "cloud-provider-config",
+				key:       "config",
+				skip:      nil,
+			}
+
+			ccmConfigMap := config_params{
+				namespace: "openshift-cloud-controller-manager",
+				name:      "cloud-conf",
+				key:       "cloud.conf",
+				skip:      nil,
+			}
+
+			section_name := "LoadBalancer"
```

Comment:
I know it's a pattern in this file, nevertheless we're starting to mix snake_case with camelCase in the project, this is awful, let's just use camelCase everywhere.

Reply:
Ah, camelCase is also preferred by golang in general: https://go.dev/doc/effective_go#mixed-caps

Reply:
I changed it all along the file. Thanks! Will do like that from now on